section1/reflection.md

```diff
@@ -2,18 +2,39 @@
+
 2. What are the data types you learned about in this section? In your own words, define each.
 
+**Strings** - used for text, such as words, or sentences and are wrapped in quotes. "Hello"
+**Numbers** - numerals, written completely by themselves, x = 2
+**Boolean** - expressions that will be deamed True or False using numbers or strings. Uses <,>,==,!=,<=,>=.
+
 3. How would you log the string `"Hello World!"` to the console?
 
+`console.log("Hello world")`
+
 4. What is/are the character(s) you would use to indicate comments in a JavaScript file? What is the purpose of a code comment?
 
+A code comment gives more description of what your code is doing or any other notes you'd want to keep with your code that will not be processed by the computer.
+For one line this `// This is a note` will indicate a code comment, or for multiple lines this `*/ These are Notes */`will create a comment.
+
 5. In your own words, what is a variable? How would you explain it to a 5 year old?
 
+In code a variable is going to store some kind of data. A variable is like a box full of things with a label on it. The variable is the box, the variable name is the label that tells you what is in the box and the data is what ever is in the box which we can change.
+
 6. Think of a site or app you use frequently. What are three variables that are probably used? Which data type would each of those variables probably hold?
 
+At my current job we use Outlook for emails. I imagine the text body of emails may have a variable of something like "body" and the input from the user would be strings. The time of each email might have a variable of "timeSent" that would be a number. The code to create the subject has some kind of Boolean since when sending an email without a subject a popup message says "Are you sure?". When there is no subject there is likely a line of code to check this and maybe outputs False causing the warning message.
+
 7. In your own words, explain what concatenation is.
 
+Concatenation is combining multiple variables in an output, such as writing a sentence by adding text and adding variables in along the way. 
+
 8. Think of a site or app you use frequently. Where do you think the developers used concatenation?
 
+Whenever recieving recipts over email there may be text with your information in it such as - Thank you, `Megan` for purchasing the `Purple Couch` that will be delivered `March 4,2021` to `4567 Raspberry Lane`. I would assume they might use concatentation to insert these everchanging variables per customer. 
+
 9. What questions do you still have about the work we've done so far? (not having a question is not an option)
+
+Interpolation and concatenation seem so similar I wonder what the differences are in using them in the future?
```

Comment:
Great question - same exact function, just 2 different ways to do the same thing. Interpolation is generally preferred by devs bc they think it's easier to type. Both are fine, just pick one and use that one consistently within your project :)